api/src/main/java/com/epam/pipeline/acl/run/RunApiService.java

```diff
@@ -267,4 +271,33 @@ public Boolean checkFreeSpaceAvailable(final Long runId) {
     public PipelineRun terminateRun(final Long runId) {
         return runManager.terminateRun(runId);
     }
+
+    @PreAuthorize(RUN_ID_EXECUTE)
```

Comment:
I think it is better to have a separate `RunScheduleApiService`

Reply:
Done